moveit_ros/hybrid_planning/hybrid_planning_manager/hybrid_planning_manager_component/include/moveit/hybrid_planning_manager/hybrid_planning_events.h

```diff
@@ -46,12 +46,17 @@ enum class HybridPlanningEvent
 {
   // Occurs when the hybrid planning manager receives a planning request
   HYBRID_PLANNING_REQUEST_RECEIVED,
-  // Indicates that the global planning action finished (Does not indicate planning success! The action response
-  // contains this information)
-  GLOBAL_PLANNING_ACTION_FINISHED,
+  // Indicates that the global planning action finished
+  GLOBAL_PLANNING_ACTION_SUCCESSFUL,
+  GLOBAL_PLANNING_ACTION_ABORTED,
+  GLOBAL_PLANNING_ACTION_CANCELED,
```

Comment:
`GLOBAL_PLANNING_ACTION_ABORTED` and `GLOBAL_PLANNING_ACTION_CANCELED` mean the same thing. Can't you combine them into one?

Reply:
Oh, I see that you did this because they have different meanings for ROS2 action servers. Ok, that's fine then.